src/utilities/fetchAndExtractZip.utils.ts

```diff
@@ -73,6 +73,31 @@ const fetchZipFromUrl = async (url: string): Promise<ArrayBuffer> => {
 	return zipData;
 };
 
+export const unpackFileZip = async (file: File | ArrayBuffer): Promise<ProcessedZipResult> => {
+	try {
+		const zip = new JSZip();
+		const content = await zip.loadAsync(file);
+		const structure = await processZipContent(content);
+
+		return { structure };
+	} catch (error) {
+		const errorMessage =
+			error instanceof Error
+				? error.message
+				: i18n.t("fetchAndExtract.uknownErrorUnpackingZip", { ns: "utilities" });
```

Comment:
Add file name to toast and to logger

Reply:
added

Reply:
missing name on line 87  in the `unpackFileZip` function

Reply:
added